easybuild/easyconfigs/l/Leafcutter/leafcutter-0.2.7-foss-2018b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PackedBinary'
+
+name = 'leafcutter'
+version = '0.2.7'
+
+homepage = 'https://github.com/davidaknowles/leafcutter'
+description = """Leafcutter quantifies RNA splicing variation using 
+short-read RNA-seq data. The core idea is to leverage spliced reads 
+(reads that span an intron) to quantify (differential) intron usage across samples.  """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+#
+# Example URL: https://github.com/davidaknowles/leafcutter/archive/v0.2.9.tar.gz
+#
+source_urls = [('https://github.com/davidaknowles/leafcutter/archive/')]
+sources = [('v%s.tar.gz' % (version))]
+
+checksums = ['0e70ebce6be4b25bfabc0d489cbefa92ddede80380b1bbf4ec3a84182d10d1bd']
+
+dependencies = [ 
+    ('SAMtools', '1.9'),
+    ('RegTools', '0.5.2', '-foss-2018b-Java-11-LTS',('dummy', '')),
```

Comment:
Dit kan niet goed zijn. Je geeft een dummy toolchain op, wat suggereerd dat er geen afhankelijkheid van een toolchain is, maar vervolgens wel foss-2018b in de naam suffix... Ik zou verwachten:
`('RegTools', '0.5.2', '-Java-11-LTS'),`
en dan erft het de foss/2018b van de regel met:
`toolchain = {'name': 'foss', 'version': '2018b'}`